### DISCUSSION TITLE
Programming via BLE

### DISCUSSION THREAD

**Discussion by WolfOptics** [BIPES/BIPES#165] [Q&A] [answered]
Hi sirs,
 Is there any method or reference about that BIPES blockly connect ESP32 by BLE?
Thanks a lot.

**Comment by rafaelaroca** [comment, marked as answer]
Yes!
Here it is:
https://rafaelaroca.wordpress.com/2021/07/21/bipes-and-web-bluetooth-on-esp32-first-working-test/

There are ready-to-use blocks on BIPES for the commands mentioned on that page.

regards
Rafael

**Reply by WolfOptics** [reply]
Dear rafaelaroca,
Thanks a lot. That was work for my desktop PC and notebook, but it is NOT work in smartphone and iPad.
Is it possible communication between smartphone and MCU in the BIPES platform?

Best Regards,
